### PR TITLE
feat: updated the document creator with a downloadable config file

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:

- added a generic template config file for user to download
- added instructions to update the generic template config file so user will be able to use it.

### CODE REVIEW

File: src/document-creator/document-creator.mdx [hunk 1]
```diff
@@ -294,3 +298,89 @@ An API key is not required to be provided for these endpoints as they will accep
...
+},
+
+```
+
+- `"documentStore"` - Update with your document store. For more infomation, click [here](./deploying-store).
+- `"location"` - Update with the Issuer's domain. For more information, click [here](https://openattestation.com/docs/advanced/identity-proofs).
+- `"logo"` - Update with the Logo's URL, to be displayed accordingly as the image above.
+- `"backgroundColor"` - Update with the desired background color. The default color will be used if color is not stated here. e.g. "#000000", "black"
+- `"titleColor"` - Update with the desired title text color. The default color will be used if color is not stated here. e.g. "#ffffff", "white"
+- `"descriptionColor"` - Update with the desired description text color. The default color will be used if color is not stated here. e.g. "#ffffff", "white"
```
Comment (simboonlong) [inline src/document-creator/document-creator.mdx:372]: are there default colors set on generic templates end? if there is wanna mention wat are the default colors?

File: src/document-creator/document-creator.mdx [hunk 2]
```diff
@@ -294,3 +298,89 @@ An API key is not required to be provided for these endpoints as they will accep
...
+
+```json
+"documentStorage": {
+  "apiKey": "<Document storage API Key>",
+  "url": "<Document storage URL>"
+}
+```
+
+- `"apiKey"` - Update with the document storage API key. For more infomation, click [here](//document-creator#document-storage-field).
+- `"url"` - Update with the document storage URL. For more infomation, click [here](//document-creator#document-storage-field).
```
Comment (simboonlong) [inline src/document-creator/document-creator.mdx:384]: links dead

File: src/document-creator/document-creator.mdx [hunk 3]
```diff
@@ -294,3 +298,89 @@ An API key is not required to be provided for these endpoints as they will accep
...
+- `"descriptionColor"` - Update with the desired description text color. The default color will be used if color is not stated here. e.g. "#ffffff", "white"
+- `"title"` - Update with the default title for the document.
+- `"remarks"` - Update with the default remarks for the document.
+
+```json
+"documentStorage": {
+  "apiKey": "<Document storage API Key>",
+  "url": "<Document storage URL>"
+}
+```
```
Comment (simboonlong) [inline src/document-creator/document-creator.mdx:381]: not sure if you wanna re-mention that this field is optional, the config file can still work without it

File: src/document-creator/document-creator.mdx (hunk 1 above)
Comment (isaackps) [reply src/document-creator/document-creator.mdx:372]: ok

File: src/document-creator/document-creator.mdx (hunk 2 above)
Comment (isaackps) [reply src/document-creator/document-creator.mdx:384]: ok

File: src/document-creator/document-creator.mdx (hunk 3 above)
Comment (isaackps) [reply src/document-creator/document-creator.mdx:381]: ok

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**